examples/js/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "../../.babelrc"
+}
```

Comment:
Babel fought me. Unless I had a `.babelrc` in each repo, I couldn't get `babel-loader` etc working properly, so I opted for just adding `.babelrc` wherever needed and just extending from the base config.

Reply:
That's typical. With our Nx monorepo we needed separate ones too.